assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Ads Main PAX setup component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createInterpolateElement,
+	Fragment,
+	useCallback,
+} from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import AdsIcon from '../../../../../svg/graphics/ads.svg';
+import SetupForm from './SetupForm';
+import SupportLink from '../../../../components/SupportLink';
+import AdBlockerWarning from '../common/AdBlockerWarning';
+import { SpinnerButton } from 'googlesitekit-components';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { CORE_LOCATION } from '../../../../googlesitekit/datastore/location/constants';
+import { ADWORDS_SCOPE, MODULES_ADS } from '../../datastore/constants';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import useQueryArg from '../../../../hooks/useQueryArg';
+import { addQueryArgs } from '@wordpress/url';
+import PAXEmbeddedApp from '../PAXEmbeddedApp';
+const { useSelect, useDispatch, useRegistry } = Data;
+
+export default function SetupMainPAX( { finishSetup } ) {
+	const [ pax, setPax ] = useQueryArg( 'pax' );
+
+	const registry = useRegistry();
+
+	const isAdBlockerActive = useSelect( ( select ) =>
+		select( CORE_USER ).isAdBlockerActive()
+	);
+	const hasAdwordsScope = useSelect( ( select ) =>
+		select( CORE_USER ).hasScope( ADWORDS_SCOPE )
+	);
+
+	const redirectURL = addQueryArgs( global.location.href, {
+		pax: 1,
+		notification: 'ads', // Arbitrary value to prevent authentication_success from surfacing.
```

Comment:
We have to pass some value, otherwise service side will attach success notification, we use this in several places across the codebase, it is the only mechanism to prevent success notification

Reply:
That shouldn't matter here since it wouldn't show during module setup anyways; this is the normal behavior which you can see after granting a scope during module activation. E.g. 
<img width="1041" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/c9b4fa29-05c0-4038-a19a-ed22196d3bdc">

Is this not the case in this scenario?

Reply:
@aaemnnosttv it is needed in this case, as Ads uses subtle notification to display success banner, and it is showing here on the redirect unless silenced via this approach

@tofumatt updated the comment

Reply:
> it is needed in this case, as Ads uses subtle notification to display success banner, and it is showing here on the redirect unless silenced via this approach

I see what you mean, however, notifications shouldn't appear during module setup regardless of this parameter, they're only expected to render on the dashboard, so we should change the condition for rendering `SubtleNotifications` since it is currently rendered unconditionally in the `Header`.